lib/fluent/supervisor.rb

```diff
@@ -1196,6 +1205,28 @@ def build_system_config(conf)
       system_config
     end
 
+    def build_additional_configurations
+      if @system_config.config_include_dir&.empty?
+        $log.info :supervisor, 'default configuration include directory was disabled'
```

Comment:
```suggestion
        $log.info :supervisor, 'configuration include directory is disabled'
```

* It would be clearer if we omitted the word `default`.
  * In this case, the feature itself is disabled.
* Using the present tense would be clearer.